pkg/migration/generate.go

```diff
@@ -835,6 +835,7 @@ func (h *Handler) handleCustomImageRegistry(cluster *corev1.StorageCluster) erro
 		MetricsCollectorProxy:      h.removeCustomImageRegistry(cluster.Spec.CustomImageRegistry, cluster.Status.DesiredImages.MetricsCollectorProxy),
 		PxRepo:                     h.removeCustomImageRegistry(cluster.Spec.CustomImageRegistry, cluster.Status.DesiredImages.PxRepo),
 	}
+	cluster.Status.Version = pxutil.GetImageTag(cluster.Spec.Image)
```

Comment:
Why do we need this? This will be automatically set by the operator when the migration starts.

Reply:
If we dont have this, the desiredImages will be overwritten after stc reconcile.

Reply:
Oh yes, that would happen.

Reply:
Good point, changed